site/migrate-mcq-to-qq.md

```diff
@@ -0,0 +1,290 @@
+
+```bash
+rabbitmqctl list_consumers queue_name channel_pid
+```
+
+### `x-cancel-on-ha-failover` for Consumers
+
+Classic mirrored queues consumers can be [automatically cancelled](https://www.rabbitmq.com/ha.html#cancellation) when a queue
+leader fails over. This can cause loss of information about which messages were sent to which consumer, and result in the same messages being sent again (duplicate messages). 
+
+Some of the cases for duplicate messages are covered by `x-cancel-on-ha-failover` and others are not. Most of the cases covered by `x-cancel-on-ha-failover` do not exist with quorum queues but those that are not covered are still there. Therefore, your application must be able to handle duplicates, which it should be able to do anyway.
+
+## <a id="mcq-features-to-remove" class="anchor" href="#mcq-features-to-remove">Mirrored Classic Queue Features that can be removed from Source Code or moved to a Policy</a>
+
+The following features don't complete any function when quorum queues are
+used. The best way to handle these features is to remove them from the source
+code, or move them to a policy.
+
+### Lazy Queues
+
+Mirrored classic queues can optionally operate in [lazy
+mode](https://rabbitmq.com/lazy-queues.html#overview). For quorum
+queues this is the only way of operation. The best way to handle the migration of these mirrored classic queues
+is to move `x-queue-mode` from the source code to a policy.
+
+### Transcient Queues
+
+[Transcient queues](https://www.rabbitmq.com/queues.html#durability) are deleted on a node/cluster boot. 
+
+The plan is to remove transcient queues in future RabbitMQ releases.
+The only option for transcient queues then will be exclusive queues. This only affects the durability of queue definitions. Messages can still be marked transient.
+
+You must make a decision about transcient queues before migration, is the content of the 
+queue important enough to get availability guarantees of quorum queues, or is it better to downgrade the transcient queue to a classic non-mirrored queue (classic mirrored queues are being removed but classic non-mirrored queues will still be available).
+
+### Exclusive queues
+
+[Exclusive queues](https://www.rabbitmq.com/queues.html#exclusive-queues) are not mirrored. An attempt to create an exclusive mirrored queue creates a non-mirrored queue so the mirroring policy is ignored.
+
+For exclusive queues that are not mirrored, you must decide whether to leave the queue as an exclusive queue or change it to a replicated queue during migration. Therefore, you must be careful not to make exclusive queue declarations with an explicit `x-queue-type: quorum` argument.
+
+## General Prerequisites before Migrating to Quorum Queues
+
+1. A minimum of 3 nodes in the RabbitMQ cluster is required (there is no reason to use quorum queues with a smaller amount of replicas).
```

Comment:
This sentence is simply not true. While we recommend a minimum of 3 nodes for highly available quorum queues, it is perfectly reasonable to use quorum queues with a single node cluster, and there are some valid circumstances under which a user may decide to do so.

Reply:
I would recommend replacing this sentence with the following:
1. A RabbitMQ cluster with an odd number of nodes (for high availability, a minimum of 3 nodes is required).

Reply:
Thanks @MirahImage I will update to this based on @mkuratczyk reply to the same information above: 

1. A RabbitMQ cluster with an odd number of nodes. A minimum of 3 nodes in the RabbitMQ cluster is required for high availability

Question though @MirahImage when you say "odd number" can that be any odd number up to what??? should we be specific here? Thanks